common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -1637,14 +1637,16 @@
   "screens.wynntils.map.focus.description2": "Shift-Left click here to center around the player.",
   "screens.wynntils.map.focus.name": "Center",
   "screens.wynntils.map.help.description1": "Ctrl to show territories",
-  "screens.wynntils.map.help.description2": "Left click on waypoint to place the compass marker.",
-  "screens.wynntils.map.help.description3": "Middle click to place the compass marker on the map.",
-  "screens.wynntils.map.help.description4": "Shift-Middle click on the map to create a waypoint.",
-  "screens.wynntils.map.help.description5": "Shift-Middle click on a waypoint to edit it.",
-  "screens.wynntils.map.help.description6": "Alt-Middle click on a waypoint to remove it.",
-  "screens.wynntils.map.help.description7": "Left click on the compass marker to remove it.",
-  "screens.wynntils.map.help.description8": "Right click to center on player.",
-  "screens.wynntils.map.help.description9": "Shift-Right click to center on the compass marker.",
+  "screens.wynntils.map.help.description10": "Right click to center on player.",
+  "screens.wynntils.map.help.description11": "Shift-Right click to center on the compass marker.",
+  "screens.wynntils.map.help.description2": "Left click on waypoint to replace the compass marker.",
+  "screens.wynntils.map.help.description3": "Shift + Left click on waypoint to place an additional compass marker.",
+  "screens.wynntils.map.help.description4": "Middle click to replace the compass marker on the map.",
+  "screens.wynntils.map.help.description5": "Shift + Middle click to place an additional the compass marker on the map.",
+  "screens.wynntils.map.help.description6": "Shift-Middle click on the map to create a waypoint.",
```

Comment:
These seems to be the same, or conflicting..?

Reply:
I just removed that part. We can still add multiple waypoints on POIs but not the map. for now.

Reply:
The reordering confused me. But it seems that shift+middle click on the map will at the same time both create a new (persisted) waypoint (the old behavior), and add a new (temporary) marker (the new behavior).

But our entire set of operations is just messed up. :-( There is no logic or consistency here... This is why I was worried this will just be messy getting into place in this PR.

Reply:
Yeah. I am just not sure if we can do anything when people are used to this already.  Maybe the right-click actions list..

Reply:
We are probably a bit stuck with this for now, yes. At some point we can probably make a more meaningful alternative, and have it as a config to keep the old and weird way. But that is a bigger change.

Did I understand correctly that now the old behavior with replacing is kept for the old "gestures", but there is a new one, and that is shift + left click on an icon to add a marker to it? (but no way to have multiple markers on random places on the map, for now)

Reply:
Yes, that is how it is.